share/metainfo/net.lutris.Lutris.metainfo.xml

```diff
@@ -26,6 +26,34 @@
   <url type="translate">https://github.com/lutris/lutris/tree/master/po</url>
   <url type="bugtracker">https://github.com/lutris/lutris/issues</url>
   <launchable type="desktop-id">net.lutris.Lutris.desktop</launchable>
+  <releases>
```

Comment:
The `<releases>` tag is already bellow, please add the `<release>` to it. You can use the 

```
appstreamcli validate --explain FILE
```

to verify if the file is OK.